roda-common/roda-common-data/src/main/java/org/roda/core/data/utils/CentralEntitiesJsonUtils.java

```diff
@@ -0,0 +1,161 @@
+package org.roda.core.data.utils;
+
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.roda.core.data.common.RodaConstants;
+import org.roda.core.data.v2.synchronization.bundle.CentralEntities;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonEncoding;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+
+/**
+ * {@author João Gomes <jgomes@keep.pt>}.
+ */
+public final class CentralEntitiesJsonUtils {
+  private static final Logger LOGGER = LoggerFactory.getLogger(CentralEntitiesJsonUtils.class);
+
+  public CentralEntitiesJsonUtils() {
+    // do Nothing
+  }
+
+  /**
+   * Write to the file the given {@link List}.
+   * 
+   * @param list
+   *          {@link List}
+   * @param path
+   *          {@link Path}
+   * @throws IOException
+   *           if some i/o error occurs.
+   */
+  public static void writeListToFile(final List<String> list, final Path path) throws IOException {
+    if (!Files.exists(path)) {
+      Files.createFile(path);
+    }
+    OutputStream outputStream = null;
+    JsonGenerator jsonGenerator = null;
+    try {
+      if (path != null) {
+        outputStream = new BufferedOutputStream(new FileOutputStream(path.toFile()));
+      }
+      final JsonFactory jsonFactory = new JsonFactory();
+      jsonGenerator = jsonFactory.createGenerator(outputStream, JsonEncoding.UTF8).useDefaultPrettyPrinter();
+      jsonGenerator.writeStartArray();
+      for (String value : list) {
```

Comment:
To avoid memory problems, instead of getting a list, you can do solr search here and write using the iterator.